server/src/models/FroshModel.js

```diff
@@ -34,65 +34,55 @@ const paymentSchema = new mongoose.Schema({
 
 const FroshSchema = new mongoose.Schema(
   {
-    birthDate: {
-      type: Date,
+    legalName: {
+      type: String,
       required: true,
     },
     pronouns: {
       type: String,
-      enum: ['Prefer Not to Say', 'he/him', 'she/her', 'they/them', 'Other'],
+      enum: ['He/Him', 'They/Them', 'She/Her', 'Other', 'Prefer not to say'],
       required: true,
     },
-    pronounsOther: {
+    pronounOther: {
       type: String,
       required: false,
     },
+    birthDate: {
+      type: Date,
+      required: true,
+    },
+    utorid: {
+      type: String,
+      required: true,
+    },
     discipline: {
       type: String,
       enum: [
         'Chemical',
-        'Civil',
-        'Electrical & Computer',
-        'Engineering Science',
         'Industrial',
-        'Materials',
         'Mechanical',
+        'Civil',
         'Mineral',
-        'Track One (Undeclared)',
+        'ECE',
+        'MSE',
+        'TrackOne',
+        'EngSci',
       ],
```

Comment:
I know the exel sheet has it in this format, but, this is not very user friendly also in the profile page its displated as `______ Engineering Student` we should switch it back to how it was since its more "proper".